### PR TITLE
Add a test/run-local.sh script run via make run for development

### PR DESCRIPTION
**Pull request by metalmatze**
I thought that may be helpful. :)

/cc @squat @kakkoyun @periklis

### CODE REVIEW

**Comment by periklis** [comment]
I don't have a special opinion on this. I am more of fan using kind, but I pass to @blockloop who used to have woes to test the promtail manifests.

**Review by kakkoyun — DISMISSED** [review]
✅

**Comment by metalmatze** [comment]
@periklis, this is mostly about developing the API itself, like literally changing its code and for that running the binary without even building containers is still the fastest, and maybe even simplest. :)

File: test/run-local.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    --tsdb.path="$(mktemp -d)"
+) &
+
+(
+  ./tmp/bin/thanos query \
+    --grpc-address=127.0.0.1:10911 \
+    --http-address=127.0.0.1:9091 \
+    --store=127.0.0.1:10901 \
+    --log.level=error \
+    --web.external-prefix=/ui/metrics/v1
```
Comment (squat) [inline test/run-local.sh:49]: I think this external prefix is not quite correct. I am finding that the static assets aren't loading correctly when I run this

**Comment by metalmatze** [comment]
UI now works properly too, when running locally. :chart_with_upwards_trend:

**Comment by squat** [comment]
there seems to be something funny with the commits in this branch as they include stuff from master but rebased. Can you take a look?

**Comment by squat** [comment]
Otherwise LGTM!

**Comment by metalmatze** [comment]
I wasn't able to push and then rebased. Something went wrong.

**Comment by metalmatze** [comment]
Rebased on master after merging #96 and now the PR seems correct.

**Review by squat — APPROVED** [review]
nice! LGTM

**Pull request merged**